tests/io_adapters_test.py

```diff
@@ -52,26 +66,26 @@ def test_out_shape(self):
     input_tensor = tf.random.normal([batch, *input_shape])
     block_size = 4
     out_adapter = adp.create_output_adapter(
-                        input_shape, block_size=block_size)
+                        input_shape, block_size = block_size)
```

Comment:
Style: According to the Python style guide, if you have a standalone assignment you want to keep spaces, but if you have a function call with a named argument (like here), spaces should not be used.

Also, in this case there should be exactly 4 spaces from the beginning of the `out_adapter = ...` line. So the first character `i` in `input_shape` should be located at the first `a` in `out_adapter`.

```suggestion
        input_shape, block_size=block_size)
```